src_files.yml

```diff
@@ -1,5 +1,9 @@
 axi_node:
+  vlog_opts: [
+    -L axi_lib
+  ]
   incdirs: [
+    ../axi/src/,
```

Comment:
This path is outside of this repository, so it restricts this repository from being used in a specific environment (i.e., the parent folder must contain an `axi` folder where presumably a compatible version of [`axi`](https://github.com/pulp-platform/axi) is checked out) and violates the principle of separation of concerns.  In isolation, this repository contains all information necessary to use it in any environment and should be usable in this way.